File: app/src/main/java/com/leilei/guoshujinfu/mylearning/util/AppLog.java
package com.leilei.guoshujinfu.mylearning.util;

import android.util.Log;

/**
 * Created by Administrator on 2017/8/1.
 */

public final class AppLog {
    public static final String LOG_TAG_ACTIVITY = "activity";
    public static final String LOG_TAG_TEST = "test";
    private AppLog(){

    }
    public static void logDebug(String tag, String msg) {
        Log.d(tag, msg);
    }
    public static void logError(String tag, String msg) {
        Log.e(tag, msg);
    }


}

File: yueting/src/main/kotlin/com/aqrlei/graduation/yueting/baselib/util/AppLog.kt
package com.aqrlei.graduation.yueting.baselib.util

import android.util.Log

/**
 * @Author: AqrLei
 *@Name MyLearning
 *@Description:
 *@Date: 2017/8/22
 */
object AppLog {
    val LOG_TAG_ACTIVITY: String = "activity"
    val LOG_TAG_FRAGMENT: String = "fragment"
    val LOG_TAG_PRESENTER: String = "presenter"
    fun logDebug(tag: String, msg: String) {
        Log.d(tag, msg)
    }
}